### PR TITLE
Native Queue Length - Learning Transport

### PR DESCRIPTION
**Pull request by mikeminutillo**

### CODE REVIEW

File: src/ServiceControl.LearningTransport/QueueLengthProvider.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                }
+            });
+
+            return Task.FromResult(0);
+        }
+
+        void UpdateStore(ILookup<string, long?> queueLengths)
+        {
+            var now = DateTime.UtcNow.Ticks;
+            foreach (var instance in endpointsHash.Keys)
```
Comment (Scooletz) [inline src/ServiceControl.LearningTransport/QueueLengthProvider.cs]: Accessing keys, locks the dictionary. Please use enumerator over the dictionary and get instance from `kvp.Key`

**Comment by Scooletz** [comment]
If I'm the one to review it, it looks good to me 👍

**Comment by tmasternak** [comment]
@mikeminutillo in order to release it we still need to tweak the SCMU right?

**Comment by mikeminutillo** [comment]
Discussed with @tmasternak over slack. We probably don't want to release support for the learning transport in SCMU until ServiceControl-prime can do that as well.

**Comment by mikeminutillo** [comment]
With a smoke test in place, this is ready to merge

**Pull request merged**